config/default.yml

```diff
@@ -783,6 +783,11 @@ PrePush:
     description: 'Running RSpec test suite'
     required_executable: 'rspec'
 
+  Minitest:
+    enabled: false
+    description: 'Running Minitest test suite'
+    command: ['bundle', 'exec', 'rake', 'test']
```

Comment:
That makes sense... I was basing this off the rspec command.

Any suggestions on how to run minitest tests (similar to rspec) on PrePush? I'm happy to submit a PR either to the README or the code. I'm just not sure how to run an arbitrary rake task on PrePush without something like this PR.